### PR TITLE
[Feature] Add chat composite demo for quickstart

### PR DESCRIPTION
**Pull request by Leah-Xia-Microsoft**
## Purpose

* Created a new folder called `ui-library-quick-start` to host both `Calling` and `Chat`
* Renamed `UILibraryQuickStart` project to `Calling`
* Added a new project `Chat` to the original `UILibraryQuickStart` workspace
* Added working demo code to launch Chat composite

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
* Get the code
* Point to local podspec by changing Podfile from 
`pod 'AzureCommunicationUICalling', '1.1.0'` to
`pod 'AzureCommunicationUIChat', :podspec => '<path_to_the_podsect>'`
* Fill in the chat variables in `ViewController` and run the app

## What to Check
Verify that the following are valid
* Pressing the Start Experience button should start and join the chat thread
* Chat operations should work as expected

### CODE REVIEW

File: ui-library-quick-start/Podfile.lock [hunk 1]
```diff
@@ -1,7 +1,7 @@
 PODS:
   - AzureCommunicationCalling (2.2.1):
     - AzureCommunicationCommon (~> 1.0)
-  - AzureCommunicationCommon (1.1.0)
+  - AzureCommunicationCommon (1.1.1)
   - AzureCommunicationUICalling (1.1.0):
     - AzureCommunicationCalling (= 2.2.1)
     - MicrosoftFluentUI/ActivityIndicator_ios (= 0.3.9)
```
Comment (palatter) [inline ui-library-quick-start/Podfile.lock:7]: We should update fluentUI to match library

File: ui-library-quick-start/README.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# iOS Quick Starts
+
+Quick starts provide a quick example on how easy it is to integrate Azure Communication Services into your own app
```
Comment (palatter) [inline ui-library-quick-start/README.md:3]:
```suggestion
Quick starts provide a quick example on how easy it is to integrate Azure Communication Services into your own app.
```

File: ui-library-quick-start/Podfile [hunk 3]
```diff
@@ -1,6 +1,18 @@
...
+target 'Calling' do
+    project 'Calling/Calling.xcodeproj'
     use_frameworks!
     pod 'AzureCommunicationUICalling', '1.1.0'
 end
+
+target 'Chat' do
+    project 'Chat/Chat.xcodeproj'
+    use_frameworks!
+    pod 'AzureCommunicationUIChat', '0.1.0-beta.1'
```
Comment (jpeng-ms) [inline ui-library-quick-start/Podfile:17]: it should be 1.0.0-beta.1

File: ui-library-quick-start/Chat/Chat/ViewController.swift [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+import UIKit
+import AzureCommunicationCommon
+import AzureCommunicationUIChat
+
+class ViewController: UIViewController {
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        let button = UIButton(frame: CGRect(x: 100, y: 100, width: 200, height: 50))
```
Comment (emlynmac) [inline ui-library-quick-start/Chat/Chat/ViewController.swift:15]: Frame / size should be set with auto layout constraints

**Pull request closed without merging**